CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 ### 0.1.8 ()
 
+### Bug Fixes
+
+* The extension wont add nonsense filed to the conf ([#101](https://github.com/Certora/vscode-certora-prover/pull/101))
```

Comment:
@NoaBablikiCertora  maybe better - "removed unnecessary flags from conf files" 
what do you think?